test/lit/passes/extract-function-table-unused.wast

```diff
@@ -0,0 +1,26 @@
+;; RUN: wasm-opt %s --extract-function=foo -S -o - | filecheck %s
+;; RUN: wasm-opt %s --extract-function --pass-arg=extract-function@foo -S -o - | filecheck %s
+;; RUN: wasm-opt %s --extract-function-index=0 -S -o - | filecheck %s
+;; RUN: wasm-opt %s --extract-function-index --pass-arg=extract-function-index@0 -S -o - | filecheck %s
+
+;; CHECK:      (module
+;; CHECK-NEXT:  (type $none_=>_none (func))
+;; CHECK-NEXT:  (import "env" "other" (func $other))
+;; CHECK-NEXT:  (export "foo" (func $foo))
+;; CHECK-NEXT:  (func $foo
+;; CHECK-NEXT:   (nop)
+;; CHECK-NEXT:  )
+;; CHECK-NEXT: )
+
+(module
+  ;; Use another function in the table, but the table is not used in the
```

Comment:
```suggestion
  ;; Use a function in the table, but the table is not used in the
```
A downside of separate lit files is that we can't have a "conversation" about their connections...